web/styles/popovers.css

```diff
@@ -1054,6 +1034,89 @@ ul.popover-group-menu-member-list {
     }
 }
 
+.popover-menu-user-header {
+    display: flex;
+    flex-flow: row nowrap;
+    gap: 7px;
+    text-align: left;
+    padding: 4px;
+
+    .popover-menu-user-avatar-container {
+        position: relative;
+        flex-shrink: 0;
+        width: 64px;
+        height: 64px;
+    }
+
+    .popover-menu-user-avatar {
+        width: 64px;
+        height: 64px;
+        border-radius: 4px;
+        background-size: cover;
+        background-position: center;
+    }
+
+    .popover-menu-user-presence {
+        position: absolute;
+        width: 8px;
+        height: 8px;
+        right: -1px;
+        bottom: -1px;
+        border: solid 1px var(--color-background);
+        border-radius: 50%;
+    }
+
+    .popover-menu-user-info {
+        width: max-content;
+    }
+
+    .popover-menu-user-full-name {
+        font-size: 18px;
+        font-weight: 600;
+        line-height: 20px;
+        color: var(--color-text-full-name);
+        word-break: break-word;
+    }
+
+    .popover-menu-user-type {
+        font-size: 14px;
+        font-weight: 400;
+        line-height: 16px;
+        color: var(--color-text-item);
+    }
+
+    .user_circle_empty {
+        background-color: var(--color-background-popover-menu);
+        border-color: hsl(0deg 0% 50%);
+    }
+}
+
+.user-email-field {
```

Comment:
Again, this class should really have `user-card-popover` in its name if it's a top-level rule.